### PR TITLE
Adding section about respectful communication

### PR DESCRIPTION
**Pull request by nacklequack**

### CODE REVIEW

File: code-of-conduct.md [hunk 1]
```diff
@@ -154,6 +154,51 @@ The Woodcrest administration reserves the right to monitor and moderate all grou
...
+- Recognize that staff members are acting in the best interest of the entire community
+
+##### 15.1.1 OOC Communication and IC Actions
+
+The guidelines for respectful communication outlined in section 15.1 primarily apply to out-of-character (OOC) interactions, such as discussions on forums, Discord, or other community platforms.
+
+However, it is important to note that in-character (IC) actions and behaviors that are disrespectful, harassing, or inappropriate towards staff members or the community will be met with appropriate in-character consequences. This may include, but is not limited to:
+- IC legal repercussions, such as fines, arrests, or imprisonment
+- IC social consequences, such as damaged reputation, loss of trust, or exclusion from certain groups or events
+- IC economic consequences, such as loss of business, contracts, or partnerships
```
Comment (Rai-Bread) [inline code-of-conduct.md:176]:
```suggestion
- IC economic consequences, such as loss of contracts, or partnerships
```
Loss of business should always be an OOC decision. I'd probably want to reword this but suggesting to think of IC economic consequences that aren't just fines

File: code-of-conduct.md [hunk 2]
```diff
@@ -154,6 +154,51 @@ The Woodcrest administration reserves the right to monitor and moderate all grou
...
+
+In cases where a resident engages in disrespectful, harassing, or inappropriate behavior towards staff or the community, the following moderation steps will be taken:
+
+1. First offense: The resident will receive a formal warning from the moderation team, outlining the specific behavior that violates the Code of Conduct and the expected corrective action.
+
+2. Second offense: The resident will be temporarily suspended from participating in Woodcrest roleplay and community events for a period determined by the moderation team, based on the severity of the offense. This suspension may include temporary removal from the Woodcrest group, restricted access to sims, or other appropriate measures.
+
+3. Third offense: The resident will be permanently banned from the Woodcrest community, including removal from the Woodcrest group and a permanent ban from all Woodcrest sims.
+
+The moderation team reserves the right to escalate the penalty to a higher level, including immediate permanent banning, for severe or egregious violations of the Code of Conduct.
```
Comment (Rai-Bread) [inline code-of-conduct.md:192]:
```suggestion
The administration team reserves the right to escalate the penalty to a higher level, including immediate permanent banning, for severe or egregious violations of the Code of Conduct.
```

**Comment by HanakoBlossom** [comment]
I forgot how to review so Imma try this first edit: that didn't work >.>

**Review by HanakoBlossom — APPROVED** [review]
I DID IT

**Comment by HanakoBlossom** [comment]
> I DID IT

ps, not enough emoji on github. I'll put in a review on that later

**Pull request merged**